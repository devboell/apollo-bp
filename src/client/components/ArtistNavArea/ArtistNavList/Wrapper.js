import styled from 'styled-components'

const Wrapper = styled.div`
  display: flex;
  justify-content: center;
  margin: 20px 10px;
  flex-basis: 15%;
`

export default Wrapper
