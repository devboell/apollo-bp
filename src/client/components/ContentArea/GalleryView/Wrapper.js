import styled from 'styled-components'

const Wrapper = styled.div.attrs({
  'data-testid': 'galleryView',
})`
`

export default Wrapper
