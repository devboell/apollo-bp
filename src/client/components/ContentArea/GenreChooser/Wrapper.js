import styled from 'styled-components'

const Wrapper = styled.span`
  display: flex;
  justify-content: flex-start;
  align-items: center;
  height: 40px;
  margin-bottom: 20px;
  border: 1px solid black;
  border-radius: 5px;
`

export default Wrapper
