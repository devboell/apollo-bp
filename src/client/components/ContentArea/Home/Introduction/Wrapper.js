import styled from 'styled-components'

const Wrapper = styled.div`
  font-size: 20px;
  padding: 10px 30px;
`

export default Wrapper
