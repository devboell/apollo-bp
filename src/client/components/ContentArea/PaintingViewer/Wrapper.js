import styled from 'styled-components'

const Wrapper = styled.div.attrs({
  'data-testid': 'paintingViewer',
})`
  display: flex;
  flex-flow: column;
  justify-content: flex-start;
  width: 100%;
  margin: auto;
`

export default Wrapper
