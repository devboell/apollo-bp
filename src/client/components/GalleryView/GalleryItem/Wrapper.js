import styled from 'styled-components'

const Wrapper = styled.section`
  display: flex;
  flex-flow: column;
  align-items: center;
  margin: 0px 20px 20px 20px;
`

export default Wrapper
