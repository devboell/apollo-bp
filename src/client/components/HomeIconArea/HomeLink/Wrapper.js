import styled from 'styled-components'

const Wrapper = styled.div`
  margin-left: 20px;
  flex-basis: 15%;
`

export default Wrapper
