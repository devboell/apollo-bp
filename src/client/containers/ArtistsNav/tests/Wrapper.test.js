import React from 'react'
import 'jest-styled-components'

import Wrapper from '../Wrapper'

it('Wrapper, snapshot', () => {
  const wrapper = shallow(<Wrapper />)
  expect(wrapper).toMatchSnapshot()
})
