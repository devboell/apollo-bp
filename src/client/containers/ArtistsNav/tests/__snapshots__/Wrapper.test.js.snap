// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wrapper, snapshot 1`] = `
.gLboLS {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin: 20px 10px;
  -webkit-flex-basis: 15%;
  -ms-flex-basis: 15%;
  flex-basis: 15%;
}

<div
  className="sc-bdVaJa gLboLS"
/>
`;
